onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -269,10 +269,33 @@ def aten_angle(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_any(self: TensorType) -> TensorType:
+@torch_op("aten::any", trace_only=True)
+def aten_any(self: BOOL, dim: Optional[int] = None, keepdim: bool = True) -> BOOL:
     """any(Tensor self) -> Tensor"""
 
-    raise NotImplementedError()
+    minus_1 = op.Constant(value_ints=[-1])
+    self_rank = op.Size(op.Shape(self))
+    if self_rank == 0:
+        self = op.Reshape(self, minus_1)
+
+    self = op.Abs(self)  # change negative to positive then do op.Greater()
+    zero = op.Constant(value_float=0.0)
+    result = op.Greater(self, zero)
```

Comment:
Why do we need this step?

Reply:
Because test case has both positive number and neg number. Abs to convert them all to positive numbers, then using Greater to check if it is zero(False) or positive numbers(True).